src/renderers/shared/stack/reconciler/ReactRef.js

```diff
@@ -56,20 +66,21 @@ ReactRef.shouldUpdateRefs = function(prevElement, nextElement) {
   // is made. It probably belongs where the key checking and
   // instantiateReactComponent is done.
 
-  var prevEmpty = prevElement === null || prevElement === false;
-  var nextEmpty = nextElement === null || nextElement === false;
-
   return (
     // This has a few false positives w/r/t empty components.
```

Comment:
I think the idea is that if both `prevElement == null || prevElement === false` and `nextElement == null || nextElement === false` then the function will return true. This is a false positive because if it was empty and stays empty, there is no need to update refs. Probably not a big deal.